interpreter/syntax/ast.ml

```diff
@@ -60,6 +62,15 @@ type 'a memop =
 type loadop = (Memory.mem_size * Memory.extension) memop
 type storeop = Memory.mem_size memop
 
+module RmwOp =
```

Comment:
Is it possible to get rid of this by directly returning a rmwop from the lexer?

Reply:
Done.